ehr/server/app/config/auth.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    secret: ""
-  };
\ No newline at end of file
+  secret: "1234"

```

Comment:
We never want to include sensitive information in files we add to GitHub, even if it is just credentials for local or development testing. Here, you would want to put something similar to: 
secret: "ADD SECRET HERE"